resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/ClientSuite.scala

```diff
@@ -213,6 +214,10 @@ class ClientSuite extends SparkFunSuite with Matchers {
     }
     appContext.getMaxAppAttempts should be (42)
     appContext.getPriority.getPriority should be (1)
+    if (isApplicationTimeoutAvailable) {
+      val maybeTimeouts = ResourceRequestTestHelper.getApplicationTimeouts(appContext)
+      assert(maybeTimeouts.get === 12345L)
+    }
   }
```

Comment:
it would be nice to have an actual test that times out from this. Can we add one to YarnClusterSuite?